src/main/frontend/handler/export.cljs

```diff
@@ -482,6 +492,26 @@
                   (.setAttribute anchor "download" path)
                   (.click anchor))))))))))
 
+(defn export-page-as-opml!
+  [page-name]
+  (when-let [repo (state/get-current-repo)]
+    (when-let [file (db/get-page-file page-name)]
+      (when-let [path (:file/path file)]
+        (when-let [content (get-page-content page-name)]
+          (let [names [page-name]
+                format (f/get-format path)
+                files [{:path path :content content :names names :format format}]]
+            (let [files (export-files-as-opml repo files)]
+              (let [data (js/Blob. [(second (first files))]
+                                   (clj->js {:type "text/plain;charset=utf-8,"}))]
+                (let [anchor (gdom/getElement "export-page-as-opml")
+                      url (js/window.URL.createObjectURL data)
+                      opml-path (string/replace path #"(.+)\.(md|org)$" "$1.opml")]
```

Comment:
The file could be `.markdown` or `.Markdown`, we need to handle lower/upper case too.